spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/FirestoreTemplate.java

```diff
@@ -57,6 +58,8 @@
  * @since 1.2
  */
 public class FirestoreTemplate implements FirestoreReactiveOperations {
+	@Autowired
```

Comment:
Since FirestoreTemplate is autoconfigured, we should probably stick to adding fields through constructor or setter from in `GcpFirestoreAutoConfiguration`. This feels optional (although what happens if it's not set, an NPE? Might want to provide a default), so using `setClassMapper()` makes sense.